psi4/src/read_options.cc

```diff
@@ -1592,8 +1592,6 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
                         "ROBUST TREUTLER NONE FLAT P_GAUSSIAN D_GAUSSIAN P_SLATER D_SLATER LOG_GAUSSIAN LOG_SLATER NONE");
         /*- Do reduce numerical COSX errors with overlap fitting? !expert -*/
         options.add_bool("COSX_OVERLAP_FITTING", true);
-        /*- Do allow for improved COSX screening performance by constructing the Fock matrix incrementally? !expert -*/
-        options.add_bool("COSX_INCFOCK", false);
```

Comment:
I'll double-check this with testing; but I'm pretty sure that, since incfock_ and related variables are not used in any cc file for JK except the integral-direct ones (DFJLinK, DirectJK, DFJCOSK), it should just be ignored.

Reply:
Just checked this! Adding `set incfock true` to the beginning of the scf5 test results in no errors; all tests still run and pass fine! So it does seem that other JK methods simply ignore the INCFOCK keyword.